hw/ip/rv_plic/data/rv_plic.hjson

```diff
@@ -17,7 +17,7 @@
     { name: "NumSrc",
       desc: "Number of interrupt sources",
       type: "int",
-      default: "32",
+      default: "55",
```

Comment:
rv_plic directory is for exemplar. Could you please roll this back to 32 interrupts?

Reply:
sure will revert this.